slices/ovn-common.yaml

```diff
@@ -0,0 +1,25 @@
+package: ovn-common
+
+slices:
+  core:
+    essential:
+      - coreutils_bins
+      - sed_bins
+      - dash_bins
+      - libc6_libs
+      - libcap-ng0_libs
+      - libssl3_libs
+      - libunbound8_libs
+    contents:
+      /usr/share/ovn/scripts/ovn-ctl:
+      /usr/share/ovn/scripts/ovn-lib:
+      /var/run/ovn/: {make: true, mode: 02775}
+      /var/log/ovn/: {make: true, mode: 02775}
+      /var/lib/ovn/: {make: true, mode: 02775}
```

Comment:
Are these required? I checked out the package maintainer scripts and didn't notice anything similar.